BUILD

```diff
@@ -122,8 +122,6 @@ PYTHON3.update({
 NODEJS = {
     "{REGISTRY}/{PROJECT_ID}/nodejs:14": "//nodejs:nodejs14_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs:14-debug": "//nodejs:nodejs14_debug_amd64_debian11",
-    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:latest": "//nodejs:nodejs16_amd64_debian11",
```

Comment:
oh interesting, I guess the old config was double pushing these tags... not a real bad issue, but whoops.

Reply:
That's why I added 18 there, but wasn't exactly sure. I think the manifests overwrote this